plugins/utils/better-escape.nix

```diff
@@ -0,0 +1,65 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.better-escape;
+  helpers = import ../helpers.nix {inherit lib;};
+in {
+  options.plugins.better-escape =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "better-escape";
+
+      package = helpers.mkPackageOption "better-escape" pkgs.vimPlugins.better-escape-nvim;
+
+      mapping = helpers.mkNullOrOption (with types; listOf str) ''
+        List of mappings to use to enter escape mode.
+      '';
+
+      timeout = helpers.defaultNullOpts.mkInt 100 ''
+        The time in which the keys must be hit in ms. Use option timeoutlen by default.
+      '';
+
+      clearEmptyLines = helpers.defaultNullOpts.mkBool false ''
+        Clear line after escaping if there is only whitespace.
+      '';
+
+      keys =
```

Comment:
I believe this could also be a list of keys as well, how should I type this ? I tried `listof str`

Reply:
No, it is indeed either a string or a lua function.